<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Amrit_Prashant" failures="2" tests="3" name="Test" time="55.631" errors="0" timestamp="2023-09-13T20:38:38 IST">
  <testcase classname="script.DemoTest" name="testA" time="2.001"/>
  <testcase classname="script.DemoTest2" name="testB" time="0.02">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at script.DemoTest2.testB(DemoTest2.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testB -->
  <testcase classname="script.ValidLogin" name="testValidLogin" time="18.739">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at script.ValidLogin.testValidLogin(ValidLogin.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testValidLogin -->
</testsuite> <!-- Test -->
